shared/domain/src/main/java/com/ivy/domain/model/AccountStats.kt

```diff
@@ -0,0 +1,17 @@
+package com.ivy.domain.model
+
+data class AccountStats(
+    val income: StatSummary,
+    val expense: StatSummary,
+    val transfersIn: StatSummary,
+    val transfersOut: StatSummary,
```

Comment:
There should be a balance variable.
preferably balanceInBaseCurrency also

Reply:
I prefer this `AccountStats` to be currency-agnostic so they can be tranformed based on different needs (e.g. we can have an UI where we show the income and expenses in each currency). In follow-up PRs I'll create additional use-cases that `AccountStats -> SingleCurrencyAccountStats`. The balance itself can be later computed. My idea is our data models to not carry duplicated or ambiguous info.

Balance is duplicate or ambiguous because:
- balance = income + transfers-in - expense - transfers-out
- balance = income - expense

Reply:
SingleCurrencyAccountStats 
Might not be enough because We need balance in account currency and balanceInBasecurrecy because we show the summation of all the accounts values in the UI screen

Reply:
Good call! My idea is to split the entire logic into small composable pieces that we can recompose together into different shapes. Also the naming aren't great but can't come up with better